### ISSUE TITLE
Training Results

### ISSUE THREAD

**Issue by anabia-alam** [aws-samples/amazon-sagemaker-pytorch-detectron2#11]
Hello, 
I wanted to ask about the plotting of training loss, are only starting and ending losses plotted? I trained the model on a custom dataset, and got this result which is an incredibly straight downward slope. 
Also, is there a way to incorporate/set epochs in the training process. 
![image](https://user-images.githubusercontent.com/97160469/168285699-73118e76-103c-4693-8de8-202c0e93aff8.png)

TIA

**Comment by paoloirrera** [comment]
Hello, no the algorithm plots loss values at a regular period, which is the defined by the argument `--log-period` (default to 40). Instead of looking at the graph, you could also look at the CloudWatch logs from where you can find the values that the are used to generate these plots.
As for the second question, Detectron2 uses the notion of iteration and not of epochs. You can quite simply obtain the number of epochs as it follows `MAX_ITER * BATCH_SIZE / TOTAL_NUM_IMAGES`. So if you want to set the number of epochs, you have to adjust `--num-iter` in function of batch_size and the total number of images.

**Comment by paoloirrera** [comment]
FYI, here a plot that I got from this a while ago:
![image](https://user-images.githubusercontent.com/75016840/168292563-bedbf00d-76a0-4cb1-a565-0d218d778dc6.png)

**Comment by anabia-alam** [comment]
Thanks for the response. 
Also wanted to ask, does the code base implement early stopping to prevent over-fitting?

**Comment by paoloirrera** [comment]
No, it doesn't.
